user/create_interactor.go

```diff
@@ -2,7 +2,7 @@ package user
 
 // CreateUserInputBoundary represents an interactor that creates a User
 type CreateUserInputBoundary interface {
-	Handle(input createUserInputData, presenter *createUserOutputBoundary)
+	Handle(input createUserInputData, presenter createUserOutputBoundary)
```

Comment:
https://gobyexample.com/interfaces

https://stackoverflow.com/questions/44370277/type-is-pointer-to-interface-not-interface-confusion